components/Accordion.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+interface AccordionItem {
+  question: string;
+  answer: string;
+}
+
+interface AccordionProps {
+  items: AccordionItem[];
+}
+
+const Accordion: React.FC<AccordionProps> = ({ items }) => {
+  const [activeIndex, setActiveIndex] = useState<number | null>(null);
+
+  const handleToggle = (index: number) => {
+    setActiveIndex((prevIndex) => (prevIndex === index ? null : index));
+  };
+
+  return (
+    <div>
+      {items.map((item, index) => (
+        <div
+          key={index}
+          className={`overflow-hidden transition-max-height border-t-2 ${
+            activeIndex === index ? 'max-h-96' : 'max-h-20'
+          }  ${index === items.length - 1 ? 'border-b-2' : ''}`}
+        >
+          <div
+            className='flex justify-between items-center p-4 cursor-pointer'
+            onClick={() => handleToggle(index)}
+          >
+            <div className='text-[20px]'>{item.question}</div>
```

Comment:
For questions to have their own externally accesible url well need to add a self link something like:

````
<div className='text-[20px]'><a href="#{question-id}">{item.question}</a></div>
````

Reply:
What I did in this is that on clicking of question the accordion expands and then show the answer of the quesiton but with this addition it will redirect to other link. 
So what I did in this is to add arrow in the end of answer on clicking of that arrow they can redirect to respected links. 
Let me know about your views on this

Reply:
The accordion expansion effects works well but we need to also change the url from /faq to something like  /faq#xxxxxxxxxx. The blue arrow you added I think doesn't add any value. Just focus on adding the self link navigation when expanding the accordion.

Reply:
Got it I'll update this